Tests/LibWeb/Text/input/DOM/Elements-from-point-2.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<style>
+    .box {
+        width: 100px;
+        height: 100px;
+        position: absolute;
+    }
+
+    #a {
+        background-color: magenta;
+        z-index: 1;
+        transform: translate(110px, 10px);
+    }
+
+    #b {
+        background-color: mediumaquamarine;
+        z-index: 2;
+        transform: translate(120px, 20px);
+    }
+
+    #c {
+        background-color: greenyellow;
+        z-index: 3;
+        transform: translate(130px, 30px);
+    }
+</style>
+<div id="a" class="box"></div>
+<div id="b" class="box"></div>
+<div id="c" class="box">hello</div>
+<script src="../include.js"></script>
```

Comment:
Is this test similar to the one from mdn? https://developer.mozilla.org/en-US/docs/Web/API/Document/elementsFromPoint#html

On master that doesn't give us the `<p>` element in either API. I guess I'm asking to add that example as another test :P

Reply:
yes, this example does from with changes in this branch. I added it as another test.